using-valtimo/plugin/smtp-mail/configure-smtp-mail-plugin.md

```diff
@@ -0,0 +1,53 @@
+# SMTP Mail Plugin: A Simple Guide
```

Comment:
Going to leave a general comment here as when I go through each thing individually I might as well make the changes myself. The PR does not adhere to our style guidelines. Please follow https://docs.valtimo.nl/readme/style-guide.

Second: Please look at other using valtimo pages for inspiration on what the descriptions should be. As it stands this page kind of sticks out like a sore thumb. I know you generated this, so my suggestion would be to either give it more information, or write it yourself.

Reply:
@marijnritense i completely rewrote it :D Let me know if sth is still not correct